inc/namespace.php

```diff
@@ -76,6 +80,17 @@ function generate_docs_for_module( $id, Module $module ) : ?Group {
 
 	$group = new Group( $module->get_title() );
 
+	return add_docs_for_gruop( $group, $doc_dir );
+}
+
+/**
+ * Generate documentation for a module.
+ *
+ * @param string $id Module ID.
+ * @param Module $module Module object.
+ * @return Group Documentation group for the module.
```

Comment:
Type should be `Group|null`

Reply:
Can nullable type hints be used? eg `?Group`